sda-commons-server-opentracing-example/src/test/java/org/sdase/commons/server/opentracing/example/OpenTracingApplicationIT.java

```diff
@@ -39,6 +39,12 @@ void shouldDoError() {
     assertThat(response.getStatus()).isEqualTo(500);
   }
 
+  @Test
+  void shouldDoException() {
+    Response response = webTarget("/exception").request().get();
+    assertThat(response.getStatus()).isEqualTo(200);
```

Comment:
As it is a tracing test, should we verify the spans? Then the tests actually makes a little sense.

Reply:
Would you change the example to use a `MockTracer` to access the last finished spans? This is how testing is done in our opentracing module.

Reply:
Probably like that. I guess, we do not have an affordable setup for a real integration test.